packages/streams/timestamp-throttle.js

```diff
@@ -32,6 +32,15 @@ function TimestampThrottle(options) {
     options && options.getMilliseconds
       ? options.getMilliseconds
       : getMilliseconds
+  this.adjustTimestamp = false
+  if (options) {
+    if (options.adjustTimestamp === true) {
+      this.adjustTimestamp = adjustTimestamp
```

Comment:
I don't see how this can work, as it is referencing global, undefined varilable `adjustTimestamp` and not `options.adjustTimestamp`

You can achieve the same end result with just `this.adjustTimestamp = options && this.adjustTimestamp`.